website/docs/r/cloud_provider_access.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: mongodbatlas_cloud_provider_access"
+sidebar_current: "docs-mongodbatlas-datasource-cloud-provider-access"
+description: |-
+    Provides a Cloud Provider Access settings resource register, authorization, and deauthorization
```

Comment:
slight edit:      Provides a Cloud Provider Access settings resource for registration, authorization, and deauthorization